source/scripts/Provider/EthereumProvider.ts

```diff
@@ -343,6 +345,11 @@ export class EthereumProvider implements IRpcChainRequestHandler {
         delete data.types['EIP712Domain'];
       }
 
+      const activeChainId = ETH_NETWORK[activeNetwork.Ethereum].chainId;
```

Comment:
How is this error being handled in the UI?

Reply:
We're handling this error on the wallet demos site. I left a screenshot in the description that shows the error message.